hapi/src/config/index.js

```diff
@@ -5,5 +5,7 @@ const massiveDB = (async () => await massive(massiveConfig))()
 
 module.exports = {
     massiveConfig,
-    massiveDB
+    massiveDB,
+    defaultContractScope: process.env.HAPI_RATING_CONTRACT,
+    edenContractScope: 'genesisdeden'
```

Comment:
pending to add to .env

correct contract scope is `genesis.eden` but could create that account in jungletest so a created another as similar as possible and it was `genesisdeden`